patches/proxy-uri.diff

```diff
@@ -139,6 +146,37 @@ Index: code-server/lib/vscode/src/vs/code/browser/workbench/workbench.ts
 +
 +			// If not localhost, return unmodified
 +			return Promise.resolve(resolvedUri)
++		},
++		tunnelProvider: {
++			tunnelFactory: (tunnelOptions: TunnelOptions, tunnelCreationOptions: TunnelCreationOptions) => {
++				const onDidDispose: Emitter<void> = new Emitter();
++				let isDisposed = false;
++				return Promise.resolve({
++					remoteAddress: tunnelOptions.remoteAddress,
++					//The complete local address(ex. localhost:1234)
```

Comment:
nit: we might not need this comment, the code I think makes it obvious it is the local address and the format it takes and I think "complete" in this context is ambiguous anyway (I would expect that to mean absolute and include the protocol).  I think we just copied this from elsewhere anyway.

Reply:
Yes! It was copied. I'll remove it.